src/julia_internal.h

```diff
@@ -597,7 +597,7 @@ static inline void jl_set_gc_and_wait(void)
 }
 #endif
 
-void jl_dump_native(const char *bc_fname, const char *obj_fname, const char *sysimg_data, size_t sysimg_len);
+void jl_dump_native(const char *bc_fname, const char *bc_unopt_fname, const char *obj_fname, const char *sysimg_data, size_t sysimg_len);
```

Comment:
Some unrelated changes crept in during rebase. Will take them out in the morning.